docs/content.zh/docs/dev/datastream/application_parameters.md

```diff
@@ -24,28 +24,27 @@ specific language governing permissions and limitations
 under the License.
 -->
 
```

Comment:
<a name="handing-application-parameters"></a>

please check it for other same posistions.